scripts/setup.sh

```diff
@@ -11,12 +15,19 @@ fi
 instance_ip="$(terraform output -raw instance_ip)"
 trap "echo To debug failures, run: ssh pachyderm@$instance_ip -- journalctl -n 100 -f -u pachyderm-aml-syncer" EXIT
 
-[[ -z "${TF_VAR_skip_pachyderm_deploy}" ]] && terraform output -raw kube_config > kubeconfig
+[[ -z "${TF_VAR_skip_pachyderm_deploy:+x}" ]] && terraform output -raw kube_config > out/kubeconfig
 
-cd ..
+popd
 # copy over env variables and kubeconfig
-scp -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -r $(pwd)/terraform/kubeconfig pachyderm@$instance_ip:/home/pachyderm/.kube/config
-scp -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -r $(pwd)/scripts/{env.sh,helmvalues.yaml} pachyderm@$instance_ip:/home/pachyderm
+kubeconfig=$TERRAFORM_WD/out/kubeconfig
+if [[ -f ${kubeconfig} ]]; then
+    scp -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -r ${kubeconfig} pachyderm@$instance_ip:/home/pachyderm/.kube/config
+else
+    echo "error: ${kubeconfig} not found"
```

Comment:
I think ideally we'd put this check at the beginning of setup.sh—otherwise, if a user gets this error, they'll have to copy their kubeconfig to the right directory and then re-run `setup.sh` from the beginning, which means re-doing all the terraform stuff farther up.

Reply:
(I'm basically just imagining:
```
if [[ -z "${TF_VAR_skip_pachyderm_deploy:+x}" ]] && [[ ! -f "$TERRAFORM_WD/out/kubeconfig" ]]; then
  echo "$TERRAFORM_WD/out/kubeconfig" does not exist, but Pachyderm will not be deployed; exiting" >/dev/stderr
  exit 1
fi
```
)